lagtraj/forcings/conversion/input_definitions_nudging.py

```diff
@@ -0,0 +1,60 @@
+"""
+For nudging parameters we have quite a few sets of parameters and they can be
+used to set the parameters for nudging momentum and scalar forcing profiles
+separately (for example `nudging_timescale_scalars` and
+`nudging_timescale_momentum`). To avoid typos while enumerating all those
+options (and the cases for each nudging method were each parameter is required)
+this code takes care of generating the valid combinations.
+"""
+
+
+AVAILABLE_NUDGING_METHODS = [
```

Comment:
Should we add "off", i.e. an explicit acknowledgement that nudging is turned off?

Reply:
Good idea